typescript/sdk/src/hook/CosmosNativeHookModule.ts

```diff
@@ -188,6 +188,7 @@ export class CosmosNativeHookModule extends HyperlaneModule<
       await this.signer.setIgpOwner({
         igp_id: igp.id,
         new_owner: config.owner,
+        renounce_ownership: !config.owner, // if owner is empty we renounce the ownership
```

Comment:
I feel this is incorrect here because we are going to execute this code block only if the `config.owner` field is not a falsy value `[ if(config.owner && this.signer.account.address !== config.owner) ]`  but here we are checking if it is empty, but empty in javascript is falsy so this block will never execute in the first place.

Reply:
you're right here, removed the check in the if statement